case-studies/docker/docker-case-study.md

```diff
@@ -0,0 +1,265 @@
+    <li><a href="#conclusion">Conclusion</a></li>
+    <li>
+        <a href="#getting-started-with-docker">Getting started with Docker</a>
+        <ul>
+            <li>
+                <a href="#installing">Installing</a>
+                <ul>
+                    <li><a href="#windows">Windows</a></li>
+                    <li><a href="#macos">MacOS</a></li>
+                </ul>
+                <li><a href="#use-docker">Use Docker</a></li>
+            </li>
+        </ul>
+    </li>
+    <li><a href="#references">References</a></li>     
+  </ol>
+
+## Introduction 
+
+**What is Docker Why use docker**
+
+Docker is one of the most popular containerizing technology companies, but the word Docker is can be referred to [Docker inc](https://www.docker.com/), the company it can also be use when it comes to [Docker Engine](https://docs.docker.com/engine/) this is running the virtualizing environment. most have heard of virtualizing, many will think about virtualizing of game, virtual environment. But if you have heard about [Virtual Machine](https://en.wikipedia.org/wiki/Virtual_machine) docker engine is like this but with some changes docker share the kernel with the host machine,
+This makes much faster to boot and it do not need [kernel](https://en.wikipedia.org/wiki/KERNAL) to run, Docker will then use less H/W resources to run. Docker take the application, Dependencies to run a Container and the docker engine run with [REST API](https://en.wikipedia.org/wiki/Representational_state_transfer) and [CLI](https://en.wikipedia.org/wiki/Command-line_interface) to let user connect to the container. For developer it's good development platform for building good workflow, Version Control on Software, like [NodeJS](https://en.wikipedia.org/wiki/Node.js), [Java](https://en.wikipedia.org/wiki/Java_(programming_language)), [python](https://en.wikipedia.org/wiki/Python_(programming_language)), [MariaDB](https://en.wikipedia.org/wiki/MariaDB), [SQL](https://en.wikipedia.org/wiki/SQL). but not only developer use this platform, Cyber-security resource, Cloud base Platform. Docker is good tool in the IT world today. Most Operative system, Client base and server has support for docker. today Docker has one of the biggest Container image libraries in the world on [Docker Hub](https://hub.docker.com/). 
+
+
+
+## Brief History 
+
+when docker launched in 2013 it’s, was a Linux-based kernel containerization technology. Only Linux user was using docker before docker partner up with [Redhat](https://www.redhat.com/en) and This may be the most prosperous partnership Docker has made because this led to even bigger partnership in docker history, [Microsoft](https://www.microsoft.com/) wants to make this technology ready for Windows users and Windows server. Before Microsoft made partnership with Docker [LinkedIn](https://www.linkedin.com/) so that Docker had grew by 160% present by 2016. By 2019 docker and Microsoft released docker to windows and windows server.  
```

Comment:
Please check capital letters and grammar. I recommend using Grammarly free and ChatGPT to assist with this.